### ISSUE TITLE
Exception emitted on salt-master [ERROR ] Unhandled exception running status.proxy_reconnect after junos.zeroize .

### ISSUE THREAD

**Issue by rsmekala** [Juniper/salt#100]
### Description of Issue/Question

```
root@nms5-salt-master-b:/srv/salt# salt "vmx1" junos.zeroize
[WARNING ] /usr/local/lib/python2.7/site-packages/salt/payload.py:149: DeprecationWarning: encoding is deprecated, Use raw=False instead.
  ret = msgpack.loads(msg, use_list=True, ext_hook=ext_type_decoder, encoding=encoding)

[WARNING ] /usr/local/lib/python2.7/site-packages/jnpr/junos/device.py:670: RuntimeWarning: 
CLI command is for debug use only!
Instead of:
cli('request system zeroize')
Use:
rpc.request_system_zeroize()

  warnings.warn(warning_string, RuntimeWarning)

vmx1:
    ----------
    message:
        Completed zeroize and rebooted
    out:
        True
root@nms5-salt-master-b:/srv/salt# [ERROR   ] Unhandled exception running status.proxy_reconnect
```

**Comment by rsmekala** [comment]
The exception is not streamed onto the shell session. However, the exception is seen in the logs. This is expected behavior.

Zeroize will remove the non-root user configuration as part of its functionality, hence proxy is unable to reconnect using the non-root user.

Link: https://www.juniper.net/documentation/en_US/junos/topics/reference/command-summary/request-system-zeroize.html

**Comment by rsmekala** [comment]
Closed PR: [1429667](https://gnats.juniper.net/web/default/1429667-1)
